pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -1679,6 +2037,14 @@ func TestUserSignupDeactivatedButMURDeleteFails(t *testing.T) {
 			Message: "unable to delete mur",
 		})
 	AssertThatCounterHas(t, 1)
+
+	// There should not be a notification created since the mur deletion failed even if reconciled again
+	_, err = r.Reconcile(req)
+	require.Error(t, err)
+	notifications := &v1alpha1.NotificationList{}
+	err = r.client.List(context.TODO(), notifications)
+	require.NoError(t, err)
+	require.Len(t, notifications.Items, 0)
```

Comment:
I believe that these kinds of checks could be extracted to a separated function - it makes the code more compact and shorter.

Reply:
Good suggestion, created a new function in the test/notification package for that. Done in https://github.com/codeready-toolchain/host-operator/pull/289/commits/c53daf95d071e6c1317f7f9e1ad78d0fd163e927